src/controller/CHIPDeviceController.cpp

```diff
@@ -1212,6 +1212,8 @@ CHIP_ERROR DeviceCommissioner::ConvertFromOperationalCertStatus(OperationalCrede
         return CHIP_ERROR_INCORRECT_STATE;
     case OperationalCertStatus::kTableFull:
         return CHIP_ERROR_NO_MEMORY;
+    case OperationalCertStatus::kInvalidAdminSubject:
+        return CHIP_IM_GLOBAL_STATUS(InvalidCommand);
```

Comment:
This does not seem quite right.  It loses information about the actual error....

Reply:
I suppose this depends on what the purpose of this function is. It's only used in one place. I suppose you could directly encapsulate the cluster error response as a CHIP_IM_CLUSTER_STATUS, but it looks like the purpose here is twofold:
1) So we can trigger the goto exit functionality off the error and 
2) To make the print more verbose. 

For 1, using the cluster status, or really any comparable error is fine. For 2, we could easily just print the actual cluster error and it would probably make debugging easier. The other error messages got lucky because the general pool of errors already contains a specific mapping for the cluster specific error that will print nicely. This one doesn't seem to have a direct sdk relation. I suppose we could add one - that seems to be what has happened already, but that rather negates the purpose of having a cluster specific error in the first place, if every error is part of the general pool.